controllers/volumereplicationgroup_controller.go

```diff
@@ -1776,6 +1773,26 @@ func (v *VRGInstance) updateVRGConditions() {
 	setVRGDataErrorCondition(&v.instance.Status.Conditions, v.instance.Generation, msg)
 }
 
+func (v *VRGInstance) vrgReadyStatus() {
+	if v.instance.Spec.ReplicationState == ramendrv1alpha1.Secondary {
+		v.log.Info("Marking VRG ready with replicating reason")
+
+		msg := "PVCs in the VolumeReplicationGroup group are replicating"
+		setVRGDataReplicatingCondition(&v.instance.Status.Conditions, v.instance.Generation, msg)
+
+		return
+	}
+
+	if v.instance.Spec.ReplicationState == ramendrv1alpha1.Primary {
+		v.log.Info("Marking VRG data ready after establishing replication")
+
+		msg := "PVCs in the VolumeReplicationGroup group have established replication replication"
```

Comment:
The terms `group` and `replication` appears twice in `msg`, but as I mentioned earlier, VRG doesn't yet have enough confirmation that replication has been established.  To say, `PVCs in VolumeReplicationGroup are ready`, does the VRG needs have more than one PVCs? :-) What does PVCs becoming ready mean in the secondary, because one can't write the PVC in secondary.  Hence, I wonder if the msg can be simplified to something like `VolumeReplicationGroup is ready`.  Stating that `VolumeReplicationGroup is ready as primary` may be helpful during transitions between primary and secondary.

Reply:
Change the message to "PVCs in the VolumeReplicationGroup are ready for use"

Reply:
ACK. 

Changed the message to "PVCs in the VolumeReplicationGroup are ready for use"